### DISCUSSION TITLE
Minio Configuration

### DISCUSSION THREAD

**Discussion by arlyaraujo** [devlikeapro/waha#704] [Q&A]
I am configuring a docker swarm stack to enable the use of minio. I configured it according to the documentation as shown below:

- WAHA_MEDIA_STORAGE=S3
- WAHA_S3_REGION=
- WAHA_S3_BUCKET=
- WAHA_S3_ACCESS_KEY_ID=
- WAHA_S3_SECRET_ACCESS_KEY=
- WAHA_S3_ENDPOINT=
- WAHA_S3_FORCE_PATH_STYLE=True
- WAHA_S3_PROXY_FILES=True

When I receive an audio file, it works perfectly, I tested it through webhooks in n8n, I analyzed it inside the bucket, the file stores correctly, however, when they are image or video files, it does not store them and the error appears in n8n as well, not showing the media url, and showing 404.

![image](https://github.com/user-attachments/assets/637f2783-573c-4cb1-8eea-56a76360fab4)

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! Could you share the full payload please? You can send it to waha@devlike.pro if it has some private parts :pray: 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
